### PR TITLE
Response area detail adjustments

### PR DESCRIPTION
**Pull request by halvorsanden**

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/8Bv9foYj](https://trello.com/c/8Bv9foYj)

File: layouts/partials/api/schema.html [hunk 1]
```diff
@@ -32,52 +32,64 @@
...
+              <dt class="plm mlxs">
+                <code>{{- $param -}}</code>
                 {{if eq $param $requiredParam}}{{- partial "api/required.html" -}}{{- end -}}
               </dt>
+
+              <dd>
+                {{ .description }}
+                <div class="gray text-note">{{ .type }}</div>
+              </dd>
+
```
Comment (halvorsanden) [inline layouts/partials/api/schema.html:86]: Enables/fixes this (the valid key was lacking type info and description)
Screenshot 2022-02-25 at 11 40 00

But also causes this
Screenshot 2022-02-25 at 11 38 03

**Comment by halvorsanden** [comment]
Still details and things to adjust to communicate the different levels and improve responsiveness but I think this is good enough for now and we will iterate on this next week.

**Review by Igor-J86 — APPROVED** [review]
Looks good to me! :)

**Pull request merged**